docs/concepts/consensus-protocol/negative-unl.md

```diff
@@ -16,11 +16,11 @@ The Negative UNL has no impact on how the network processes transactions or what
 
 ## Background
 
-Each server in the XRP Ledger protocol has its own UNL (Unique Node List), a list of validators it trusts not to collude, and each server independently decides when a ledger version is validated based on the consensus when enough of its trusted validators agree on a new ledger version. (The default configuration uses a recommended UNL, signed by Ripple, consisting of validators Ripple considers to be sufficiently unique, reliable, and independent.) The standard quorum requirement is at least 80% of trusted validators must agree.
+Each server in the XRP Ledger protocol has its own UNL (Unique Node List), a list of validators it trusts not to collude, and each server independently decides when a ledger version is validated based on the consensus when enough of its trusted validators agree on a new ledger version. The standard quorum requirement is at least 80% of trusted validators must agree. By default, servers are configured to download two lists, one UNL published by the XRP Ledger Foundation and one UNL published by Ripple, and to use the union of those two lists as their UNL. (In other words, by default a server listens to the votes of any validator that is on at least one of those lists.)
```

Comment:
Each server in the XRP Ledger protocol has its own UNL (Unique Node List), a list of validators it trusts not to collude. Each server independently decides when a ledger version is validated, based on the consensus when enough of its trusted validators agree on a new ledger version. 

The standard quorum requirement is at least 80% of trusted validators must agree. By default, servers are configured to download two lists: one UNL published by the XRP Ledger Foundation, and one UNL published by Ripple. Servers use the union of those two lists as their UNL. (In other words, by default, a server listens to the votes of any validator that is on at least one of those lists.)

Reply:
I ended up rewriting this a little more thoroughly:

> Each server in the XRP Ledger protocol independently decides a ledger version is validated when a quorum of its trusted validators agree. A server's trusted validators are defined by that server's UNL (Unique Node List), a list of validators it trusts not to collude. By default, servers are configured to trust any validator on at least one of two lists, one UNL published by the XRP Ledger Foundation and one UNL published by Ripple.
>
> The standard quorum requirement is 80% of trusted validators. If more than 20% of trusted validators go offline or become unable to communicate with the rest of the network, servers stop validating new ledgers because they cannot reach a quorum. This is a design choice to ensure that no transactions' outcomes can be changed after they are declared final. During such a situation, the remaining servers would still be online and could provide past and tentative transaction data, but could not confirm the final, immutable outcome of new transactions.